web-common/src/features/dashboards/rows-viewer/RowsViewerAccordion.svelte

```diff
@@ -92,26 +100,45 @@
   }
 
   $: isLocal = $featureFlags.readOnly === false;
+
+  const rowsViewerLayout = writable<LayoutElement>({
+    value: INITIAL_HEIGHT_EXPANDED,
+    visible: true,
+  });
 </script>
 
 <div>
-  <button
-    aria-label="Toggle rows viewer"
-    class="w-full bg-gray-100 h-7 text-left px-2 border-t border-t-gray-200 text-xs text-gray-800 flex items-center gap-1"
-    on:click={toggle}
+  <div
+    class="flex items-center px-2 h-7 w-full bg-gray-100 border-t border-t-gray-200 {isOpen
+      ? '!cursor-move'
```

Comment:
This should be `cursor-ns-resize` or `cursor-row-resize`

Reply:
Yeah, just switched to `cursor-ns-resize`. Originally I chose `cursor-move` to mirror the behavior of the other `use:drag` instances, but, you're right, this is more correct.